Makefile

```diff
@@ -16,6 +16,17 @@ TARGET   := alertmanager
 
 include Makefile.COMMON
 
+REV        := $(shell git rev-parse --short HEAD)
```

Comment:
Let's not depend on it being a git repo (for example, this will fail if someone downloads the automatic source zip file that GitHub generates for a release). In Prometheus, this was solved like this:

https://github.com/prometheus/prometheus/blob/master/Makefile.INCLUDE#L66-L67